controllers/front/AbstractController.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+use PagaMasTarde\ModuleUtils\Model\Log\LogEntry;
+
+/**
+ * Class AbstractController
+ */
+abstract class AbstractController extends ModuleFrontController
+{
+    /**
+     * PMT_CODE
+     */
+    const PMT_CODE = 'paylater';
+
+    /**
+     * EXCEPTION RESPONSES
+     */
+    const CC_ERR_MSG = 'Unable to block resource';
+    const CC_NO_MERCHANT_ORDERID = 'Merchant Order Id (cart_id) not found';
+    const CC_NO_CONFIG = 'Unable to load module configuration';
+    const CC_MALFORMED = 'Bad request, module may not be enabled';
+
+    const GMO_ERR_MSG = 'Unable to find merchant Order';
```

Comment:
Please explain me this constants I do not understand them

Reply:
Please export them to module utils and use them from there

Reply:
Deleted constans